js/global.js

```diff
@@ -924,9 +924,13 @@ jQuery(document).ready(function($) {
                 message = '<div class="InformMessage">' + message + '</div>';
                 // Insert any transient keys into the message (prevents csrf attacks in follow-on action urls).
                 message = message.replace(/{TransientKey}/g, gdn.definition('TransientKey'));
-                // Insert the current url as a target for inform anchors
-                message = message.replace(/{SelfUrl}/g, document.URL);
-
+                if (gdn.definition('SelfUrl')) {
```

Comment:
This will always be true, because gdn.definition returns the key if the value can't be found (as opposed to gdn.getMeta).